home/templates/home/index.html

```diff
@@ -3,10 +3,88 @@
-    <a class="btn btn-ci" href="{% url 'home' %}">Sample Button</a>
-    <br>
-    <small>custom class: <strong>btn-ci</strong></small>
+   <div class="container-fluid">
+    <div class="row">
+        <div class = "col-6 col-sm-12  col-md-6">
+            <h1 class=" text text-center">Upcoming Hackathons</h1>
+            <div id="accordion">
+            <h3>Hackathon 01 Title</h3>
+                <div>
+                    <p>
+                    Mauris mauris ante, blandit et, ultrices a, suscipit eget, quam. Integer
+                    ut neque. Vivamus nisi metus, molestie vel, gravida in, condimentum sit
+                    amet, nunc. Nam a nibh. Donec suscipit eros. Nam mi. Proin viverra leo ut
+                    odio. Curabitur malesuada. Vestibulum a velit eu ante scelerisque vulputate.
+                    </p>
+                </div>
+                <h3>Hackathon 02 Title</h3>
+                <div>
+                    <p>
+                    Sed non urna. Donec et ante. Phasellus eu ligula. Vestibulum sit amet
+                    purus. Vivamus hendrerit, dolor at aliquet laoreet, mauris turpis porttitor
+                    velit, faucibus interdum tellus libero ac justo. Vivamus non quam. In
+                    suscipit faucibus urna.
+                    </p>
+                </div>
+                <h3>Hackathon 04 Title</h3>
+                <div>
+                    <p>
+                    Nam enim risus, molestie et, porta ac, aliquam ac, risus. Quisque lobortis.
+                    Phasellus pellentesque purus in massa. Aenean in pede. Phasellus ac libero
+                    ac tellus pellentesque semper. Sed ac felis. Sed commodo, magna quis
+                    lacinia ornare, quam ante aliquam nisi, eu iaculis leo purus venenatis dui.
+                    </p>
+                    <ul>
+                    <li>List item one</li>
+                    <li>List item two</li>
+                    <li>List item three</li>
+                    </ul>
+                </div>
+                <h3>Hackathon 05 Title</h3>
+                <div>
+                    <p>
+                    Cras dictum. Pellentesque habitant morbi tristique senectus et netus
+                    et malesuada fames ac turpis egestas. Vestibulum ante ipsum primis in
+                    faucibus orci luctus et ultrices posuere cubilia Curae; Aenean lacinia
+                    mauris vel est.
+                    </p>
+                    <p>
+                    Suspendisse eu nisl. Nullam ut libero. Integer dignissim consequat lectus.
+                    Class aptent taciti sociosqu ad litora torquent per conubia nostra, per
+                    inceptos himenaeos.
+                    </p>
+                </div>
+            </div>
+        </div><!---col div-->
+    <!--carousel-->  
+        <div class = "col-6 col-sm-12  col-md-6">
+            <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">

```

Comment:
Again, let's try to give the IDs something more unique to our project, perhaps something like: `id="carousel-project-showcase"`.